template/CHART_NAME/values.yaml

```diff
@@ -445,6 +497,55 @@ ingress:
   ##
   secrets: []
 
+## Enable persistence using Persistent Volume Claims
+## ref: https://kubernetes.io/docs/user-guide/persistent-volumes/
+##
+persistence:
+  ## @param persistence.enabled Enable persistence using Persistent Volume Claims
+  ##
+  enabled: true
+  ## @param persistence.path Path to mount the volume at.
+  ##
+  path: /bitnami/%%MAIN_OBJECT_BLOCK%%/data
```

Comment:
`persistence.path` is not implemented either. 
In addition, I think both `path` and `subPath` cover the same functionality, but I think `subPath` is already present in more charts.

Reply:
They are not the same functionaility. The subpath is useful when mounting files inside directories while the path when mounting directories. I just forgot to add the invocation in the deployment/statefulset/damoenset. Fixing it.